torch/csrc/distributed/rpc/init.cpp

```diff
@@ -421,17 +421,16 @@ If the future completes with an error, an exception is thrown.
   module.def(
       "_invoke_rpc_torchscript",
       [](const std::string& dstWorkerName,
-         const py::object& userCallable,
+         const std::string& qualifiedNameStr,
          const py::tuple& argsTuple,
          const py::dict& kwargsDict) {
```

Comment:
nit: change them back to py::args and py::kwargs?

Reply:
Changing this will cause memory leak. I will re-land the rest and debug on this later.